pkg/engine/funcs.go

```diff
@@ -88,6 +90,19 @@ func toYAML(v interface{}) string {
 	return strings.TrimSuffix(string(data), "\n")
 }
 
+func toYAMLPretty(v interface{}) string {
+	var data bytes.Buffer
+	encoder := goYaml.NewEncoder(&data)
+	encoder.SetIndent(2)
```

Comment:
I think this should be configurable (but I would agree that the default should be 2)

Reply:
For what it's worth this approach is consistent with `toPrettyJson`, [documented here](https://helm.sh/docs/chart_template_guide/function_list/#toprettyjson-musttoprettyjson). This is a Sprig function call that takes no arguments as [documented here](https://masterminds.github.io/sprig/defaults.html#:~:text=.Item.-,toPrettyJson,-%2C%20mustToPrettyJson), and just prints what is conventionally considered "pretty JSON".

Reply:
oops, I meant to give an example, that any consistent indent value is allowed. e.g. this is valid yaml:

```
foo:
           - bar1
           - bar2
           - bar3
```

But also, the sprig function could be extended at some point (I think) to have a default ident of '2' if someone (not sure who :) ) would ever want a different default.